mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopNewsRow.java

```diff
@@ -49,17 +54,17 @@ public TopNewsRow(final View itemView, HomePager.OnUrlOpenListener onUrlOpenList
             public void onClick(View v) {
                 isNewsExpanded = !isNewsExpanded;
                 final int newsCount = adapter.getTopNewsCount();
+                final int changedCount = newsCount - COLLAPSED_NUM_NEWS_COUNT;
```

Comment:
What does happen if we have less than 3 news?

Reply:
It would crash. I have handled this in the next commit.